custom_components/sat/climate.py

```
@@ -822,6 +826,11 @@ async def _async_control_setpoint(self, pwm_state: PWMState) -> None:
             self._setpoint = self._calculated_setpoint
             _LOGGER.info("Pulse Width Modulation is disabled or in IDLE state. Running normal heating cycle.")
             _LOGGER.debug("Calculated setpoint for normal cycle: %.1f°C", self._calculated_setpoint)
+
+            # Some final checks to see even it's even warm
+            if self._setpoint < COLD_SETPOINT:
```

Ruff (0.14.2)

`COLD_SETPOINT` may be undefined, or defined from star imports

(F405)